### ISSUE TITLE
Uncaught (in promise) TypeError: MutationObserver.observe: Argument 1 is not an object.

### ISSUE THREAD

**Issue by ganeshnischayp** [cibernox/ember-power-select#1448]
Hi There,
Im facing this issue, any help what is causing this?

> Uncaught (in promise) TypeError: MutationObserver.observe: Argument 1 is not an object.
> Ember 2
>     invoke backburner.js:371
>     flush backburner.js:263
>     flush backburner.js:470
>     _end backburner.js:1044
>     _boundAutorunEnd backburner.js:684
>     promise callback*buildNext/< backburner.js:29
>     flush Ember
>     _scheduleAutorun backburner.js:1263
>     _end backburner.js:1054
>     _boundAutorunEnd backburner.js:684
>     promise callback*buildNext/< backburner.js:29
>     flush Ember
>     _scheduleAutorun backburner.js:1263
>     _end backburner.js:1054
>     _boundAutorunEnd backburner.js:684
>     promise callback*buildNext/< backburner.js:29
>     flush Ember
>     _scheduleAutorun backburner.js:1263
>     _end backburner.js:1054
>     _boundAutorunEnd backburner.js:684
>     promise callback*buildNext/< backburner.js:29
>     flush Ember
>     _scheduleAutorun backburner.js:1263
>     _end backburner.js:1054
>     _boundAutorunEnd backburner.js:684
>     promise callback*buildNext/< backburner.js:29
>     flush Ember
>     _scheduleAutorun backburner.js:1263
>     _end backburner.js:1054
>     _boundAutorunEnd backburner.js:684
>     promise callback*buildNext/< backburner.js:29
>     flush Ember
>     _scheduleAutorun backburner.js:1263
>     _ensureInstance backburner.js:1251
>     ensureInstance backburner.js:1002
>     <anonymous> Ember
>     dirtyTagFor validator.js:569

`{{#power-select-multiple
oninput=(action "onInputFunction")
allowClear=true
search=(action "queryFunction")
placeholder="List name or ID"
selected=ListSelected
onchange=(action "didSelectItem")
as |items|
}}
{{items.name}}
{{/power-select-multiple}}`

I am facing this issue only after building, during dev it works perfectly fine.

Clicking the dropdown causes this

Using Versions
` "ember-power-select": "^2.3.5"`
`"ember-cli": "~3.20.2"`

**Comment by ganeshnischayp** [comment]
It is duplicate of [https://github.com/cibernox/ember-power-select/issues/1122](https://github.com/cibernox/ember-power-select/issues/1122)
